src/Uno.Extensions.Templates/content/unoapp-extensions/azure-pipelines.yml

```diff
@@ -0,0 +1,217 @@
+trigger:
+- master
+- main
+- release
+
+pool:
+  vmImage: 'windows-latest'
+
+variables:
+  #[REQUIRED]
+  #For connecting your azureSubscription to the pipeline:
+  #1.- Go to your project settings (Bottom Left option)
+  #2.- Scroll down to 'Service Connections'
+  #3.- Click on 'Create new service connection' button
+  #4.- Click on 'Azure Resource Manager' and ckick on Next
+  #5.- Select ''Service Principal (automatic) and click on Next
+  #6.- Select the subscription you want to use and set a Name for your new service connection
+  #7.- Finally, use the name you defined in previous step as value for this variable
+- name: azureAccount
```

Comment:
Why is an azure account required as part of the CI pipeline, other than for publishing perhaps?

Reply:
Hey Nick i am not sure because at least in my case it is always asking me for a connected account for being able to execute the scripts. Not sure it it's because i'm using a free account.